manim/utils/space_ops.py

```diff
@@ -412,24 +636,30 @@ def shoelace_direction(x_y):
     return "CW" if area > 0 else "CCW"
 
 
-def cross2d(a, b):
+def cross2d(a: int, b: int) -> float:
```

Comment:
The typehint is wrong, it should be List, np.ndarray, List of lists, List of ndarrays, ndarray of ndarrays or ndarray or lists

Reply:
Yes this is wrong, I tired checking this function but was getting an error every time I checked.

Code:
```py
test = cross2d(np.ndarray([1,0]), np.ndarray([0,1]))
print(test)
print(type(test))
```

Output:
```cmd
IndexError                                Traceback (most recent call last)
<ipython-input-10-3ab00a0c329d> in <module>
----> 1 test = cross2d(np.ndarray([1,0]), np.ndarray([0,1]))
      2 print(test)
      3 print(type(test))

<ipython-input-2-74e7b45c05ec> in cross2d(a, b)
      1 def cross2d(a: int, b: int) -> float:
      2     if len(a.shape) == 2:
----> 3         return a[:, 0] * b[:, 1] - a[:, 1] * b[:, 0]
      4     else:
      5         return a[0] * b[1] - b[0] * a[1]

IndexError: index 0 is out of bounds for axis 1 with size 0
```